src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUtility.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.Cache.Memory;
+using Microsoft.Framework.FileSystemGlobbing;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
+{
+    /// <summary>
+    /// Utility methods for <see cref="ITagHelper"/>'s that support attributes containing file globbing patterns.
+    /// </summary>
+    public class GlobbingUtility
+    {
+        private FileProviderGlobbingDirectory _baseGlobbingDirectory;
+
+        public GlobbingUtility(IMemoryCache cache, IFileProvider fileProvider, PathString requestPathBase)
+        {
+            Cache = cache;
+            FileProvider = fileProvider;
+            RequestPathBase = requestPathBase;
+            _baseGlobbingDirectory = new FileProviderGlobbingDirectory(fileProvider, fileInfo: null, parent: null);
+        }
+
+        /// <summary>
+        /// The <see cref="IMemoryCache"/> to cache globbing results in.
+        /// </summary>
+        public IMemoryCache Cache { get; set; }
```

Comment:
Went around that already with the others. Leaving as get/set for now as they're internal.